cmd/registry/cmd/upload/artifact.go

```diff
@@ -101,6 +105,38 @@ func buildArtifact(ctx context.Context, parent string, filename string) (*rpc.Ar
 	return artifact, nil
 }
 
+func buildDisplaySettingsArtifact(ctx context.Context, jsonBytes []byte) (*rpc.Artifact, error) {
+	m := &rpc.DisplaySettings{}
+	err := protojson.Unmarshal(jsonBytes, m)
+	if err != nil {
```

Comment:
OK, I've updated that one. I'll watch for this, but note that it's inconsistently applicable - for example, the code right below it can't be collapsed:
```
	artifactBytes, err := proto.Marshal(m)
	if err != nil {
		return nil, err
	}
```
because that takes artifactBytes out of scope for the subsequent expressions.

Reply:
Right, it's not always possible so `err` sometimes needs to be function scoped. When we can though, it's good to reduce scope.